src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -384,7 +399,8 @@ export default {
 			return this.$el.querySelectorAll('li.acli_wrapper .acli')
 		},
 		setIsFocused(event) {
-			if (event.relatedTarget?.className.includes('input-field__clear-button') || this.searchText !== '') {
+			if ((event.type === 'click' && event.relatedTarget?.className.includes('input-field__clear-button'))
```

Comment:
Same here. How could I trigger a "click" event from here? We're calling this method only from `@focus` and `@blur` of SearchBox

Reply:
Changed, it was a hopeless attempt to fix the incorrect tabbing flow at root . Should be a follow-up I suppose.